tests/autopilot/autopilot_test.go

```diff
@@ -245,14 +246,19 @@ var _ = Describe(fmt.Sprintf("{%sRestartAutopilot}", testSuiteName), func() {
 			},
 		}
 
+		// Set driver namespace
```

Comment:
Get autopilot  namespace

Reply:
I'm not getting autopilot namespace here, I'm getting driver namespace.

Reply:
Set this for TODO